package.json

```diff
@@ -53,9 +53,13 @@
   ],
   "license": "(Apache-2.0 WITH LLVM-exception)",
   "bugs": {
-    "url": "https://github.com/bytecodealliance/jco/issues"
+    "url": "https://github.com/nomicfoundation/jco/issues"
+  },
+  "homepage": "https://github.com/nomicfoundation/jco#readme",
```

Comment:
If we're going to expose our fork as the landing site of the package, it would be good to add information to the readme exposing that this is a fork from bytecodealliance, and what are the reasons for it. This is to help lost souls that got this package to know where to place issues.

Reply:
I would like to avoid any unnecessary edits/changes to upstream files if possible.
I added a note about this in the homepage. WDYT?